.github/pull_request_template.md

```diff
@@ -28,14 +28,3 @@
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-
-## Checklist:
```

Comment:
Are we removing the PR checklist?

Reply:
For this upgrade repository, I feel it doesn't make any sense, except maybe the first point.

Reply:
Ah, never mind I wasn't thinking about this being a different repo 😅